x/crisis/internal/keeper/integration_test.go

```diff
@@ -0,0 +1,42 @@
+package keeper_test
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+	dbm "github.com/tendermint/tm-db"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func createTestApp() *simapp.SimApp {
+	db := dbm.NewMemDB()
+	app := simapp.NewSimApp(log.NewNopLogger(), db, nil, true, 5)
+	// init chain must be called to stop deliverState from being nil
+	genesisState := simapp.NewDefaultGenesisState()
+	stateBytes, err := codec.MarshalJSONIndent(app.Codec(), genesisState)
+	if err != nil {
+		panic(err)
+	}
+
+	// Initialize the chain
+	app.InitChain(
+		abci.RequestInitChain{
+			Validators:    []abci.ValidatorUpdate{},
+			AppStateBytes: stateBytes,
+		},
+	)
+	app.Commit()
+	app.BeginBlock(abci.RequestBeginBlock{Header: abci.Header{Height: app.LastBlockHeight() + 1}})
+
+	return app
+}
+
+func testPassingInvariant(_ sdk.Context) (string, bool) {
```

Comment:
what are these for? they don't seem to provide any value other than its name

Reply:
they are used in `keeper_test.go`. I believe it is just there for readability of a passing/failing invariant function

Reply:
can we delete them then? 🙏